packages/main/cypress/specs/Card.cy.tsx

```diff
@@ -1,24 +1,171 @@
+import Button from "../../src/Button.js";
 import Card from "../../src/Card.js";
 import CardHeader from "../../src/CardHeader.js";
 
+function CardSample() {
+	return <Card id="card">
+		<CardHeader
+			id="cardHeader"
+			slot="header"
+			additionalText="4 of 10"
+			titleText="Quick Links"
+			subtitleText="Quick links sub title"
+			interactive={true}>
+		</CardHeader>
+		<div></div>
+	</Card>;
+}
+
+function CardSample2() {
+	return <Card id="card2">
+		<CardHeader
+			id="cardHeader2"
+			slot="header"
+			additionalText="4 of 10"
+			titleText="Quick Links"
+			subtitleText="Quick Links">
+		</CardHeader>
+		<div></div>
+	</Card>;
+}
+
+describe("Card general interaction", () => {
+	it("Tests initial rendering", () => {
+		cy.mount(<CardSample />);
+
+		// assert
+		cy.get("#card").should("exist");
```

Comment:
This is not a "real" test - please remove. This test can never fail. Furthermore, if the card does not render at all, none of the follow-up tests will work.